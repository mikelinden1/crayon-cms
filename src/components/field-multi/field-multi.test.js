import FieldMulti from './field-multi';

it('FieldMulti matches snapshot', () => {
    const props = {
        name: 'test-field',
        allValue: ['val1', 'val2', 'val3'],
        actions: {
            setItemProp: () => {},
            setMultiItemProp: () => {}
        }
    };

    const wrapper = shallow(<FieldMulti {...props} />);
    expect(wrapper).toMatchSnapshot();
});