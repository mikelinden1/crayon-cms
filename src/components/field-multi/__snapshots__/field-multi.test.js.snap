// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldMulti matches snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-md-9"
    >
      <Field
        actions={
          Object {
            "setItemProp": [Function],
            "setMultiItemProp": [Function],
          }
        }
        allValue={
          Array [
            "val1",
            "val2",
            "val3",
          ]
        }
        name="test-field"
        onChange={[Function]}
      />
    </div>
    <div
      className="col-md-3"
    >
      <Button
        className="btn-block"
        color="success"
        onClick={[Function]}
        tag="button"
      >
        Add
      </Button>
    </div>
  </div>
  <Connect(FieldMultiValue)
    name="test-field"
  />
</div>
`;
